### PR TITLE
Added support for origin in start scripts.

### PR DESCRIPTION
**Pull request by kwoodson**
Adding support for openshift origin in start scripts for master and node.

### CODE REVIEW

File: parts/k8s/openshiftmasterscript.sh [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 # TODO: /etc/dnsmasq.d/origin-upstream-dns.conf is currently hardcoded; it
 # probably shouldn't be
+SERVICE_TYPE=origin
+if [ -f "/etc/sysconfig/atomic-openshift-node" ]; then
```
Comment (kargakis) [inline parts/k8s/openshiftmasterscript.sh]: @jim-minter this got me thinking that we should enable both modes in acs-engine, right?

Comment (kargakis) [reply parts/k8s/openshiftmasterscript.sh]: Ah no, disregard. This is based on the node image.

**Comment by kargakis** [comment]
/lgtm
overall

/hold
for a second approval from @jim-minter / @pweil-

File: parts/k8s/openshiftmasterscript.sh (hunk 1 above)
Comment (kwoodson) [reply parts/k8s/openshiftmasterscript.sh]: @kargakis, masters are nodes so I felt like this was safe.  If we have a better way of determining origin vs openshift-enterprise I'd be happy to use that.

Other ideas?
- Checking rpm installation (rpm -qa | grep 'origin-')
- Should we check if we are on CentOS?

**Pull request merged**